src/main/windows/mainWindow.ts

```diff
@@ -107,9 +107,11 @@ function createMainWindow(options: {linuxAppIcon: string}) {
     mainWindow.once('ready-to-show', () => {
         mainWindow.webContents.zoomLevel = 0;
 
-        mainWindow.show();
-        if (windowIsMaximized) {
-            mainWindow.maximize();
+        if (Config.hideOnStart === false) {
```

Comment:
Minor nitpick
```suggestion
        if (!Config.hideOnStart) {
```

Reply:
I prefer explicit type checking instead of inverting it or type casting it. And I like the code the same as its meaning. May be this will be better:

```typescript
if (Config.hideOnStart !== true)
```

If config if not set to `true`, then show window.

Reply:
> May be this will be better:

if (Config.hideOnStart !== true)

if we are going to be explicit, I prefer the original version. I'm ok with leaving it as it is, as I said it was a nitpick as it is my preference, but no big deal.